src/lib/client/JSKafkaClient.ts

```diff
@@ -1,14 +1,30 @@
 "use strict";
 
-const uuid = require("uuid");
-const { Kafka, PartitionDrainer, Drainer, Publisher } = require("sinek");
-const debug = require("debug")("kafka-streams:jsclient");
-
-const KafkaClient = require("./KafkaClient.js");
+import uuid from 'uuid';
```

Comment:
Actions failing due to: Drainer expecting 5 options, even though some of them should be optional: https://github.com/nodefluent/node-sinek/blob/16395fefbe37cca09d751dafd421040bd4834288/index.d.ts#L356

https://github.com/nodefluent/node-sinek/blob/16395fefbe37cca09d751dafd421040bd4834288/lib/kafka/Drainer.js#L12

Reply:
Have fixed this as part of: https://github.com/nodefluent/kafka-streams/pull/166